### PR TITLE
V4.4.1 rhel backports

### PR DESCRIPTION
**Pull request by vrothberg**
Backport of https://github.com/containers/podman/pull/17451 to pass gating.

Backport of https://github.com/containers/podman/pull/17541 to get CI to work at all (@cevich PTAL).

```release-note
None
```

### CODE REVIEW

**Comment by vrothberg** [comment]
@jnovy PTAL

**Comment by jnovy** [comment]
@vrothberg I can't comment on Chris' AWS amendments but Ed's cherry pick is much desirable and LGTM!

**Review by lsm5 — APPROVED** [review]
/lgtm
/hold

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

**Pull request merged**
